core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -71,7 +71,7 @@
       <span class="oppia-navbar-breadcrumb-separator"></span>
       <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header oppia-exploration-header" itemprop="description">{{exploration_title}}</span></h1>
     </li>
-    <li ng-click="showInformationCard()" tooltip="<['I18N_PLAYER_INFO_TOOLTIP' | translate]>" tooltip-placement="bottom" style="cursor: pointer;" class="protractor-test-exploration-info-icon" tabindex="0">
+    <li ng-click="showInformationCard()" tooltip="<['I18N_PLAYER_INFO_TOOLTIP' | translate]>" tooltip-placement="bottom" style="cursor: pointer;" class="protractor-test-exploration-info-icon exploration-info-icon" tabindex="0">
```

Comment:
oppia-exploration-info-icon

(as a general rule, any local CSS class is generally namespaced to oppia, in order to prevent conflicts with third-party libs)